### ISSUE TITLE
Needs M5Cardputer Library to compile

### ISSUE THREAD

**Issue by wcg** [aat440hz/SSHClient-M5Cardputer#2]
I had to add the M5Cardputer library (https://github.com/m5stack/M5Cardputer) in order to get this to compile.

**Comment by TinyuZhao** [comment]
This project demonstrates how to create an SSH terminal using an M5Cardputer device.

**Comment by CommentedNewline** [comment]
> I had to add the M5Cardputer library (https://github.com/m5stack/M5Cardputer) in order to get this to compile.

exactly. its a project for the m5cardputer, so you need the library. just so you know, this is not what github issues are for.
